_includes/admin-company-overview.md

```diff
@@ -0,0 +1,16 @@
+
+A company provides a single point of visibility across multiple organizations. This view simplifies the management of Docker organizations and settings. It's available to Docker Business subscribers.
+
+The following diagram depicts the set up of a company and how it relates to associated organizations. 
+
+![company-process](/admin/images/docker-hierarchy.svg){: width="700px" }
+
+## Key features
+
+With a company, administrators can:
+
+- View and manage all nested organizations and configure settings centrally.
```

Comment:
@ajthilakan @craig-osterhout do we also want to mention User management specifically?

Reply:
It may be easier to see in the generated content. User management is included below this section. This particular source file doesn't contain it because it's included/shared with Hub.
https://deploy-preview-17549--docsdocker.netlify.app/admin/company/